specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -506,6 +596,232 @@
           }
         }
       }
+    },
+    "/weather/tropical/active/{format}": {
+      "get": {
+        "description": "**Get Active Storms**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nGet all government-issued active storms.",
+        "operationId": "Weather_GetActiveStorms",
+        "x-ms-examples": {
+          "GetActiveStorms": {
+            "$ref": "./examples/GetActiveStorms.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/WeatherApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ActiveStormResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/{format}": {
+      "get": {
+        "description": "**Search government-issued Storms**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nSearch government-issued storms by year, basin ID, and government ID.",
+        "operationId": "Weather_StormSearch",
+        "x-ms-examples": {
+          "StormSearch": {
+            "$ref": "./examples/GetStormSearch.json"
+          },
+          "StormSearch with three conditions": {
+            "$ref": "./examples/GetFullStormSearch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/WeatherApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          },
+          {
+            "$ref": "#/parameters/WeatherYear"
+          },
+          {
+            "$ref": "#/parameters/WeatherBasinID"
+          },
+          {
+            "$ref": "#/parameters/WeatherGovernmentID"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StormSearchResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/forecasts/{format}": {
+      "get": {
```

Comment:
We need to somehow make WeatherYear, WeatherBasinID, WeatherGovernmentID as required here.
Just want to confirm does adding `required: ["year", "basinId", "govermentId"]` work here?

Reply:
I have added the optional and required basinId and governmentId to have the required fields.